onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -298,6 +298,14 @@ def test_set_providers_with_options(self):
             self.assertEqual(option["trt_engine_cache_path"], str(engine_cache_path))
             self.assertEqual(option["trt_force_sequential_engine_build"], "1")
 
+            from onnxruntime.capi import _pybind_state as C
+
+            session_options = C.get_default_session_options()
+
+            # TRT plugins registered as custom op domain should only be added once in session option regaldless of number of session creation
```

Comment:
typo: regardless.

suggest you actually keep the sessions around
 sess1 = onnxrt.InferenceSession(...)
and sess 2 = onnxrt.InferenceSession(...)

Reply:
just a thought: i guess there's no way to read back what ops/trtplugins were registered implicitly. maybe that would be a nice api to have?

Reply:
> typo: regardless.
> 
> suggest you actually keep the sessions around sess1 = onnxrt.InferenceSession(...) and sess 2 = onnxrt.InferenceSession(...)

Fixed

Reply:
> just a thought: i guess there's no way to read back what ops/trtplugins were registered implicitly. maybe that would be a nice api to have?

Currently there is no such API, but yes it's good to have. Probably can add it in another PR.